blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/astyanax/AstyanaxWriter.java

```diff
@@ -277,6 +277,7 @@ public void insertMetrics(Collection<IMetric> metrics, ColumnFamily cf) throws C
                                 metric.getMetricValue(),
                                 (AbstractSerializer) (Serializers.serializerFor(metric.getMetricValue().getClass())),
                                 metric.getTtlInSeconds());
+                        Instrumentation.markFullResPreaggregatedMetricWritten();
```

Comment:
This method ```insertMetrics()``` is being called for writes to other ColumnFamilies as well, because Column Family is a parameter to this method. You should probably check and make sure Column Family is metrics_preaggregated_full, before calling the ```Instrumentation.markFullResPreaggregatedMetricWritten```.